components/brave_wallet_ui/utils/api-utils.ts

```diff
@@ -29,7 +31,7 @@ export const GetFlattenedAccountBalances = (accounts: WalletAccountType[], userV
         .map(account => {
           const balance = token.contractAddress
             ? account.tokenBalanceRegistry[token.contractAddress.toLowerCase()]
-            : account.balance
+            : account.nativeBalanceRegistry[token.chainId || '']
```

Comment:
does nativeBalanceRegistry haver a key of `''`?

Reply:
Actually removed this since `chainId` is not optional anymore.